I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 331, in script
    python:
  File "game/script.rpy", line 332, in <module>
    for symbol in current_code:
NameError: name 'current_code' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 331, in script
    python:
  File "C:\Users\kt201\RenPy\renpy-8.2.1-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\kt201\RenPy\renpy-8.2.1-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 332, in <module>
    for symbol in current_code:
NameError: name 'current_code' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.2.1.24030407
Observatoria 1.0
Mon Apr  8 21:15:33 2024
